packages/components/spacings/spacings-inset-squish/README.md

```diff
@@ -36,10 +36,11 @@ import Spacings from '@commercetools-uikit/spacings';
 
 ## Properties
 
-| Props      | Type             | Required | Values            | Default |
-| ---------- | ---------------- | :------: | ----------------- | ------- |
-| `scale`    | `String`         |    -     | `['s', 'm', 'l']` | `m`     |
-| `children` | `PropTypes.node` |    -     | -                 | -       |
+| Props                   | Type             | Required | Values            | Default |
+| ----------------------- | ---------------- | :------: | ----------------- | ------- |
+| `scale`                 | `String`         |    -     | `['s', 'm', 'l']` | `m`     |
+| `useAllAvailableHeight` | `boolean`        |    -     | `true`, `false`   | `false` |
```

Comment:
Nice. This is very explicit and nicely constrained as an API. Did you think about going with e.g. `height` being a enum of `expand` or `collapse` to avoid boolean props and be more open towards future changes?

Reply:
Yes, I was hesitant about using a `boolean` here, but I didn't think we would need to have more values for the new property. However, I like the property name better so let me quickly change it.

Reply:
Totally agree. I also don't see us adding more values (famous last works 😉). I like that booleans are slightly shorter (no value in positive case) also. Thanks for taking on the improvement to the same though. Maybe it's clearer on that front and hence over all better.

Reply:
Updated here: ce034c3fba22c83026e6dc72f71a21e05df063f8